src/include/catalog/pg_namespace.h

```diff
@@ -123,7 +123,7 @@ DESCR("Standard public schema");
 DATA(insert OID = 6104 ( "pg_aoseg" PGUID _null_ 0));
 DESCR("Reserved schema for Append Only segment list and eof tables");
 #define PG_AOSEGMENT_NAMESPACE 6104
-
+#define PG_INFORMATION_SCHEMA_NAMESPACE 10671
```

Comment:
Is the oid for information_schema schema fixed?  I thought it was created during hawq init procedure, so the oid might be not fixed at the value 10671.   Can we get it firstly as a global variable by running "select oid from pg_namespace where nspname = 'information_schema'"  ?